src/rules/tripleEqualsRule.ts

```diff
@@ -22,44 +22,38 @@ export class Rule extends Lint.Rules.AbstractRule {
     public static EQ_FAILURE_STRING = "== should be ===";
     public static NEQ_FAILURE_STRING = "!= should be !==";
 
-    public apply(syntaxTree: TypeScript.SyntaxTree): Lint.RuleFailure[] {
-        return this.applyWithWalker(new ComparisonWalker(syntaxTree, this.getOptions()));
+    public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+        var comparisonWalker = new ComparisonWalker(sourceFile, this.getOptions());
+        return this.applyWithWalker(comparisonWalker);
     }
 }
 
 class ComparisonWalker extends Lint.RuleWalker {
-    public visitBinaryExpression(node: TypeScript.BinaryExpressionSyntax): void {
-        var position = this.positionAfter(node.left);
+    private static COMPARISON_OPERATOR_WIDTH = 2;
 
+    public visitBinaryExpression(node: ts.BinaryExpression) {
         if (!this.isExpressionAllowed(node)) {
-            this.handleOperatorToken(position, node.operatorToken);
+            var endOfLeftExpression = node.left.getEnd();
+            this.handleOperatorToken(endOfLeftExpression + 1, node.operator);
```

Comment:
this is a bit of a hack, but there's not a whole lot we can do given that `node.operator` no longer has position info, it's just a `SyntaxKind`